framework/docs/src/1_abstract_platform_overview.md

```diff
@@ -0,0 +1,72 @@
+<!-- This page is a high-level overview of the Abstract Platform, not the SDK or any of our actual products. Hence we should stick to a vocabulary that is familiar to the reader. -->
+![SDK Background](https://github.com/AbstractSDK/assets/blob/mainline/twitter_cover-1.png?raw=true)
+<!-- 
+<div align="center">
+
+### Welcome to Abstract, the interchain CosmWasm development platform. </br>We are building tools and infrastructure for composable smart-contract applications. 
+
+</div> -->
+## What Is The Abstract Platform?
+
+The Abstract platform embodies a comprehensive ecosystem with a set of products that empowers you to effortlessly construct, 
+deploy, and manage applications in the Cosmos ecosystem in record time.
+
+Our products are designed to be composable, allowing you to re-use the components you need to build your applications. While 
+Abstract aims to simplify the development experience, it functions as a powerful tool, enabling you to innovate with less effort.
+
+```admonish info
+In this page you are given an overview of the Abstract platform and its products. You can skip the introduction and go 
+straight to the [Getting Started](./4_get_started/1_index.md) guide if you're already familiar with the platform.
+```
+
+## Why Build with Abstract?
+
+Building decentralized applications is hard! We know this because we've been there. We've spent countless hours building 
+applications on the Cosmos ecosystem, and we've created Abstract with all the lessons we learned along the way to make it 
+easier and faster to build for you.
+
+- **Based on CosmWasm** 🌟: Abstract is built on top of the [CosmWasm][1] smart-contract framework, which has been battle-tested 
+  and proven to be secure and reliable. This also means that you can leverage the existing tooling and community to 
+  accelerate your development process.
+
+
+- **Chain-Agnostic** 🌐: The Abstract platform is built to support various blockchain networks within the Cosmos ecosystem, giving 
+  you the flexibility to choose the best-suited platform for your dApp.
+
+
+- **Modular Design** 🧩: Abstract's modular architecture allows you to choose from a curated library of smart-contract or 
+  develop your own using the Abstract SDK, enabling rapid development and customization. Think of Abstract as a lego set for 
+  building decentralized applications, connect the pieces you need and voilà!.
+
+
+- **Custom Governance** 🗳️: Abstract allows you to create custom governance structures tailored to your application's 
+  specific requirements, including integration with multi-signature wallets and DAOs.
+
+
+- **Development Tooling** 🛠: Abstract offers a vast integration testing library, enabling you to quickly increase test coverage 
+  and ensure the reliability of your dApps.
+
+
+- **Version Management** 🔄: Abstract simplifies the process of managing smart-contract versioning, ensuring you can quickly 
+  release and access new features and bug-fixes.
```

Comment:
do you feel that we should include some links or examples? This version management is possible through the registry, which I'm not sure if we explain elsewhere.